build/commands/lib/config.js

```diff
@@ -200,6 +228,15 @@ Config.prototype.enableCDMHostVerification = function () {
   return enable
 }
 
+Config.prototype.usePGO = function () {
+  // Disable PGO for non-public builds (i.e. local or PR's builds).
+  if (!this.isBraveReleaseBuild())
+    return false
+
+  // Enable PGO only for public nightly builds.
```

Comment:
why only for nightly?

Reply:
Discussed with @iefremov and enabled for all release channels.